.github/workflows/suggest_release.yml

```diff
@@ -36,4 +36,12 @@ jobs:
                 issue_number: context.issue.number,
                 labels: ['new release'],
               });
+              await github.issues.createComment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.issue.number,
+                body: |
+                  This pull request will automatically trigger a release on merge!
+                  If it shouldn't, remove the [new release] label.
```

Comment:
Won't this "`[new release]`" be detected as markdown?

Reply:
No, it doesn't have () after it

Reply:
I thought it will be treated as a reference link, but it doesn't matter much